src/tests/ftest/dfuse/multi_user.yaml

```diff
@@ -0,0 +1,11 @@
+hosts:
+  test_servers: 1
+  test_clients: 1
+pool:
+  size: 1GiB
+container:
+  type: POSIX
+  control_method: daos
+dfuse:
```

Comment:
@ashleypittman You'll need a `server_config` entry like this: https://github.com/daos-stack/daos/blob/4e8ed034053684a054881b5f0518b89ed4995847/src/tests/ftest/datamover/dst_create.yaml#L5-L16

Reply:
And `server_config` should go between `hosts` and `pool`